keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -636,9 +641,11 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
 
 * **How does this feature react if the API server and/or etcd is unavailable?**
 
+It's possible for the CPI to set the new Conditions and or Port Status in the Load Balancer Service after creating the cloud load balancer, but this feature will not be triggerable without API server response.
```

Comment:
What happens if the API server goes down after such a load balancer is already created?